lib/pgtime.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+require 'tzinfo'
+
+class PGTime
```

Comment:
👍 on breaking this stuff out into its own class. Does `PG` refer to `PagerDuty`? I tend to see it abbreviated as `PD`, but I'm not sure.

Reply:
lol, you're absolutely right. I'll fix that.

Reply:
Done.